### PR TITLE
Create firestore-rules-issue.md

### PR DESCRIPTION
**Pull request by anismemon**
Issue description

### CODE REVIEW

**Comment by doub1ejack** [comment]
@anismemon Thanks so much for putting this together.  I pushed a couple commits to add some comments and also some markdown styling.  FWIW, I find it helpful to use the preview button when reading markdown:

File: firestore-rules-issue.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+On Sunday 5/18/2020 we discovered two issues: 
+
+1. most of the data from the firebase api was suddenly no longer being imported into the PROD version of the GreenUp app. 
+
+2. simultaneously, no one with admin permissions was able to log in to the PROD version of the Greenup Admin tool.
+
+# The cause:
+
+Ultimately we determined that the root cause was that a deployment intended to deploy the admin tool also accidentally deployed breaking changes to the firestore rules.
```
Comment (doub1ejack) [inline firestore-rules-issue.md:13]: > _"intended to deploy the admin tool also accidentally deployed breaking changes to the firestore rules"_

Is this accurate?

Comment (doub1ejack) [reply firestore-rules-issue.md:13]: @nfloersch & @anismemon If one of you can weigh in on this great.  If not, I'm comfortable with this staying in and merging this PR into QA & master.

**Review by doub1ejack — APPROVED** [review]
Despite being the last one to make changes.... I approve this PR 😁

File: firestore-rules-issue.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+At first and for a long time, it was not clear whether the two issues were related, although we noted that similar permissions error logs were being generated from both the Admin tool login screen and the App api get requests.
+
+We further noted that both the Admin tool and a new set of firestore-rules had been deployed on the same day a few days previously.
+Our hypotheses centered on config/ api key issues, admin / authorization issues related to the service accounts and the Firebase Admin SDK, and firestore-rules.
+
+We quickly dropped the api keys from consideration due largely to the fact that incorrect config files would prevent **_any data_** at all from being visible.
+
+We created a new Admin SDK; using it however did not affect any behaviors.
+
```
Comment (iritush) [inline firestore-rules-issue.md:27]: We created a new Firebase Admin SDK service account. The theory behind this was based on comparing the QA and Production environments. The Admin SDK service account in QA was the newer type of service account while PROD had the older version. The one that uses keys.
**Note**: This service account that was created in production is still the active one (the old one is still there but is disabled).

Comment (iritush) [reply firestore-rules-issue.md:27]: @doub1ejack @anismemon

**Pull request merged**